en/docs/Learn/DesignAPI/Endpoints/EndpointSecurity/basic-auth.md

```diff
@@ -1,16 +1,15 @@
-# Basic Auth
+# Secure Endpoint with Basic Auth
 
-When you create an API using the API Publisher, you can specify the endpoint of the API backend implementation in the **Endpoint** tab as Production and Sandbox endpoints.
-
-If this endpoint is secured, there is an option for you to set the Auth type, credentials for the endpoint and endpoint certification under **General Endpoint Configurations.**
+When you [create an API](../../../../Learn/DesignAPI/CreateAPI/create-a-rest-api.md) using the API Publisher, you can specify the endpoint of the API backend implementation in the **Endpoint** tab as Production and Sandbox endpoints.
```

Comment:
Use {{base_path}}/Learn/DesignAPI/CreateAPI/create-a-rest-api.md

Reply:
fixed in f9ecbc5